src/view/frontend/templates/ff/asn.phtml

```diff
@@ -26,7 +26,7 @@
                 <div data-container="showMore">
 
                     <!-- set the padding on an inner element so we don't suffer from height 0 + padding issues-->
-                    <span class="text">Show More</span>
+                    <span class="text"><?= __('Show More') ?></span>
```

Comment:
Do you mind adding escaping the output with $block->escapeHtml()?

Reply:
done